docs/release-notes/highlights-7.0.0.asciidoc

```diff
@@ -60,9 +60,10 @@ image::images/highlights-7.0.0-dark-theme.png[Kibana dark theme]
 [float]
 === Responsive dashboards
 
-As part of the many enhancements to {kib} dashboards in 7.0, they are also 
-now responsive. If you want to view and interact with {kib} 
-on the go, you can pull it up on your phone with this new responsive layout.
+experimental[]As part of the enhancements to {kib} dashboards in 7.0, we've also made the design
```

Comment:
```suggestion
experimental[] As part of the enhancements to {kib} dashboards in 7.0, we've also made the design
```
You might consider adding a space after `experimental[]`. If you don't, it wont render correctly when Kibana switches to asciidoctor:
<img width="307" alt="Screen Shot 2019-03-27 at 2 59 02 PM" src="https://user-images.githubusercontent.com/5618806/55115203-29a2c280-50a1-11e9-8565-e22f5378cccc.png">